reference/toaztr.v1.yaml

```diff
@@ -1326,6 +1390,116 @@ paths:
       security:
         - toaztr_auth:
             - cases/write
+  '/v1/billing/cases/{caseId}/plans-parameters/{planId}/simulations/{simulationId}':
```

Comment:
To be removed

Reply:
Removed

Reply:
It should be a GET /bills with search parameters (filter[simulationId], etc..), this path is in my opinion too long and does not make sense here because there is no resource.

Reply:
Ah ok I see, I removed it, as well as simulationId then.